beacon-chain/blockchain/chain_info.go

```diff
@@ -328,6 +329,13 @@ func (s *Service) HeadValidatorIndexToPublicKey(_ context.Context, index types.V
 	return v.PublicKey(), nil
 }
 
+// IsOptimistic returns true if the current head is optimistic.
+func (s *Service) IsOptimistic(ctx context.Context) (bool, error) {
+	s.headLock.RLock()
+	defer s.headLock.RUnlock()
+	return s.cfg.ForkChoiceStore.Optimistic(ctx, s.head.root, s.head.slot)
```

Comment:
Should we check only this head OR we test every tip of synced_tips to be optimistic and then say optimistic?

Reply:
Only the head